tests/test_messages.py

```diff
@@ -40,6 +40,22 @@ def test_fee_info():
     assert isinstance(deserialized_message, FeeInfo)
 
 
+def test_request_paths():
+    message: Dict = dict(
+        message_type='RequestPaths',
+        token_network_address='0x82dd0e0eA3E84D00Cc119c46Ee22060939E5D1FC',
+        target_address='0x82dd0e0eA3E84D00Cc119c46Ee22060939E5D1FC',
+        value=1000,
+        num_paths=1,
+        chain_id=1,
+        nonce=1,
+        signature='signature'
+    )
+
+    deserialized_message = Message.deserialize(message)
```

Comment:
It'd be nice to add serialization test as well.
https://github.com/palango/raiden-libs/blob/19daf210185fc866e3eb3eb1902b27da1f222fa8/tests/test_messages.py#L79